broker/plugins/kubejobs/plugin.py

```diff
@@ -233,6 +235,70 @@ def errors(self):
             return ()
         return self.rds.lrange("job:errors", 0, -1)
 
+    def validate(self, data):
+        data_model = {
+            "cmd": list,
+            "config_id": six.string_types,
```

Comment:
we no longer require config_id. in the past, the Manager used to deal with attestation too, so it was required, but that's not the case anymore. it should be removed.

Reply:
Should a issue about removing everything related to config_id be created?

Reply:
Done